thundra/plugins/invocation/invocation_plugin.py

```diff
@@ -41,7 +41,7 @@ def before_invocation(self, data):
             'coldStart': InvocationPlugin.IS_COLD_START,
             'timeout': False,
             'region': self.common_data[constants.AWS_REGION],
-            'memorySize': int(context.memory_limit_in_mb)
+            'memorySize': int(getattr(context, 'memory_limit_in_mb', None))
```

Comment:
Correct me if I misunderstood this,

if context does not have a "memory_limit_in_mb" field getattr returns None
`'memorySize': int(getattr(context, 'memory_limit_in_mb', None))`

but int(None) will give an error in anyway, right? 

> TypeError: int() argument must be a string or a number, not 'NoneType' 

I assume you did this part in case of context does not have "memory_limit_in_mb" field to not get an error or exception but you couldn't achive this goal?